### ISSUE TITLE
Feature Request, use Task Scheduler by default to "autostart"

### ISSUE THREAD

**Issue by jefflord** [RamonUnch/AltSnap#475]
Why not just always use this? Was it deemed not worth it? 

For most "dev" type people elevated is must.

**Comment by RamonUnch** [comment]
I understand the advantage, however I do not want to be one of those program that has an autostart that is hidden in the task scheduler.

I rather prefer to have my program listed in the Startup tab of the task manager, so it can clearly be seen by the user if he chooses to handle its startup programs in a centralized and nice fashion like it should be.

I thought about adding another option but it would be confusing, So for now the simplest solution is to run the command line to add the scheduled task, should not be hard for a dev. Also the AltSnap.xml file can be used to setup the task.

What I will do however is to add batch files in AtSnap's folder, to add/remove AltSnap scheduled task

**Comment by jefflord** [comment]
I totally agree with the mixed feelings about programs that use Task Scheduler. 

I think it's sad that Task Manager does not see these as Autoruns can show, enable and disable them. 

Thanks for confirming, I do agree with you. And I also LOVE that you mention the issue with the current method AND the solution in a message box. 

I think only issue is the discoverability and next steps, because I did not know there was already an "AltSnap.xml". I don't think it's mentioned in the message the user sees. It only says "...or use the Task Scheduler". I created it from scratch because I did not know "AltSnap.xml" was (almost) ready for me to use/import. 

If you: 

1. Create some batch or method to help with this, including using the correct/current path to AltSnap (not placeholder in current xml) as you are suggesting and...
2. Update the message that tells the user that this stuff exists, I would be a happy camper. 

Also, a nice to have, on the weirdness of Task Scheuler and it being used by "normal apps"... When I see this I do like when the program uses a new folder for itself to make it simpler to find and then a "normal" name. E.g.:

```schtasks /create /xml .\AltSnap.xml /tn "RamonUnch-AltSnap\Login Auto Start"```
and
```<URI>\RamonUnch-AltSnap\Login Auto Start</URI>```

![image](https://github.com/RamonUnch/AltSnap/assets/4396667/4fa4de69-555f-47d4-b2b3-0ae77256f8f4)




Thanks!
